### PR TITLE
Disable renovate bot automerge

### PR DESCRIPTION
**Pull request by jphechter**
## Description

Disable Renovate bot auto merging.

## Code author checklist

- [ ] I have performed a self-review of my code
- [ ] I have created and/or updated relevant documentation, if necessary
- [ ] I have not used any relative imports
- [ ] I have checked for and removed instances of unused config from CMS
- [ ] I have pruned any instances of unused code
- [ ] I have not added any markdown to labels, titles and button text in config
- [ ] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [ ] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

**Review by cbrunefearless — APPROVED** [review]
LGTM, look at the documentationf or this.

File: renovate.json [hunk 1]
```diff
@@ -10,11 +10,6 @@
   "ignoreDeps": ["node", "cimg/node", "kill-port", "semantic-release"],
   "rebaseWhen": "conflicted",
   "packageRules": [
-    {
-      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
-      "automerge": true,
-      "automergeStrategy": "rebase"
-    },
```
Comment (aloverso) [inline renovate.json:17]: I think we should add a new rule. Not disable it across the board

**Review by aloverso — CHANGES_REQUESTED** [review]

**Comment by aloverso** [comment]
```
{
      "matchPackageNames": ["@aws-amplify/core", "@aws-amplify/auth"],
      "automerge": false
    },
```

File: renovate.json (hunk 1 above)
Comment (aloverso) [reply renovate.json:17]:
```
{
  "matchPackageNames": ["@aws-amplify/core", "@aws-amplify/auth"],
  "automerge": false
},
```

**Pull request merged**